config/locales/en.yml

```diff
@@ -626,6 +629,9 @@ en:
       one: '%{count} requested account pending.'
       other: '%{count} requested accounts pending.'
     requested_accounts_alert_view: 'View'
+    requested_account_status:
```

Comment:
Doing this just one place and re-using the same i18n message for both paths is better.

Reply:
Done!